### ISSUE TITLE
Wrong close bracket in apim.token.revocation

### ISSUE THREAD

**Issue by Sumudu-Sahan** [wso2/docs-apim#2514]
**Description:**
the close bracket of the apim.token.revocation is wrong in the documentation [1]. This should be,
```
[apim.token.revocation]
notifier_impl="org.wso2.carbon.apimgt.notification.TokenRevocationNotifier"
```

[1] https://apim.docs.wso2.com/en/latest/install-and-setup/setup/advance-configurations/configuring-caching/#clearing-the-api-gateway-cache

**Comment by isharac** [comment]
Fixed with https://github.com/wso2/docs-apim/pull/2526/commits/568843d69446eac7d9e0554cb7dc361fa193f4ee

**Comment by isharac** [comment]
fix in 3.2.0 space https://github.com/wso2/docs-apim/pull/2525
